.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: quentinleroy
+
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+<!--
+Steps to reproduce the behavior
+
+If possible add a minimal reproducer code sample
+- when using the Java client: https://github.com/zeebe-io/zeebe-test-template-java
```

Comment:
?

Reply:
Example template taken from https://github.com/camunda/zeebe/issues/new/choose

Reply:
Ok. But for me it does not make sense here. On the contrary: if people want to raise bugs for Zeebe, they should go to the Zeebe repo. So why are we linking to a zeebe template here?

Reply:
This is an example, and if it is confusing, we can change it. I'd love to hear your suggestions :)

Reply:
This can be about different components. So it might be code, it might be UX steps. Therefore I would just state something like:

> Steps to reproduce the behavior. Can be code or UI steps.

Reply:
Understood 👍 thank you for the feedback!